coding-the-legislation/35_periods.md

```diff
@@ -1,13 +1,55 @@
-# Periods
+# Periods and instants
 
 A period can be a month, a year, `n` successive months, `n` successive years or the eternity.
+The smallest unit for OpenFisca periods is the **month**. Therefore:
 
+- All periods are presumed to start on the first day of their first month.
+- A period cannot be smaller than a month.
+- To manipulate periods (e.g. create a new period), you will need to import the `periods` module like this :
```

Comment:
In which concrete legislation writing case do users need to import that? 

Unless there is a clear answer, this is blocking for me. See [previous comment](https://github.com/openfisca/openfisca-doc/pull/160#discussion_r214574658).

Reply:
I'm taking it out